ultimarc/devices/mini_pac.py

```diff
@@ -365,7 +365,7 @@ def _create_message_(self, config_file, cur_device_config=None):
                         elif alternate_action in macro_dict:
                             data.bytes[alternate_action_index] = macro_dict[alternate_action]
                         elif alternate_action:
-                            _logger.info(_(f'{pin.name} alternate action "{alternate_action}" is not a valid value'))
```

Comment:
this name variable shouldn't be changed to product_name.

Reply:
Fixed.